openam-ui/openam-ui-ria/src/main/js/org/forgerock/openam/ui/common/components/TabComponent.js

```diff
@@ -47,10 +47,10 @@ define([
             if (_.isEmpty(options.tabs)) {
                 throw new TypeError("[TabComponent] \"tabs\" argument is an empty Array.");
             }
-            _(options.tabs)
-                .each(_.partial(has, "id"))
-                .each(_.partial(has, "title"))
-                .value();
+            _(options.tabs).each(function (tab) {
+                has(tab, "id");
+                has(tab, "title");
+            });
```

Comment:
`each` immediately runs the sequence chain, so the two `each` methods here cannot be chained anymore.